src/dblib/unittests/timeout.c

```diff
@@ -162,14 +162,6 @@ test(int per_process)
 		exit(1);
 	}
 
-	dbsettime(60);
-
-	/* Verify setting a global timeout won't affect custom per-process timeouts */
-	if (dbproc->tcp_socket->query_timeout != timeout_seconds) {
-		printf("dbsettime() should not change per-process timeouts\n");
-		exit(-1);
-	}
-
```

Comment:
This hunk is removing an hunk added in the first patch. Can you remove from both?